tasklist/els-schema/src/main/java/io/camunda/tasklist/schema/manager/OpenSearchSchemaManager.java

```diff
@@ -75,7 +75,7 @@ public void createIndexLifeCycles() {
     LOGGER.info("Creating ISM Policy for deleting archived indices");
 
     final Request request =
-        new Request("PUT", "_plugins/_ism/policies/" + TASKLIST_DELETE_ARCHIVED_INDICES);
+        new Request("PUT", "/_plugins/_ism/policies/" + TASKLIST_DELETE_ARCHIVED_INDICES);
```

Comment:
I was getting `400 Bad request` from AWS ELB because of the missing slash

Reply:
actually I am getting `409 Confict` in `SchemaStartup` when `camunda.tasklist.archiver.ilmEnabled=true` and the policy already exists. I will fix it

Reply:
fixed here https://github.com/camunda/camunda/pull/18924/commits/6415ccf95ec97ee942bf8a139d3b3dcb6428f22f 
@nathansandi @marcosgvieira can you check it out?

Reply:
Yes, it is because we were checking "null" and the method now is returning  `Optional.empty()`

Still possible check on ILMPolicy class, but just update the null on that case to use empty instead.